blogposts/2022/release-3.37.md

```diff
@@ -0,0 +1,58 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/
+    category: API
+  - description: Fixed a race condition in the precise code intel upload expiry process that prematurely expired new uploads.
+    url: https://github.com/sourcegraph/sourcegraph/pull/30546
+    category: Code Intelligence
+  - description: 'Fixed performance issue in LSIF upload processing, reducing the latency between uploading an LSIF index and accessing precise code intel in the UI.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/31143
+    category: Code Intelligence
+  - description: 'Removed `experimentalFeature.showCodeMonitoringTestEmailButton`. Test emails can still be sent by editing the code monitor and expanding the "Send email notification" section.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/29953
+    category: Code Monitors
+---
+
+Sourcegraph 3.37 is now available! For this release, we introduced:
+
+## Code Intelligence just got a whole lot faster
+
+We're pleased to announce that LSIF index processing just got a whole lot faster! Earlier, certain LSIF indexes, especially for Javascript and Typescript, could take over 30 minutes to process. This meant that a long queue would build up with frequent uploads, such as those from a CI job that runs on every commit on a development branch. Now, most indexes should be processed in a few seconds instead of minutes so that you can access precise code intelligence soon after uploading an index.
+
+<img class="blog-image" title="LSIF upload speedup" alt="Comparative chart before and after processing improvements." src="https://storage.googleapis.com/sourcegraph-assets/blog/3.37/lsif-upload-speedup.png">
+
+The symbols sidebar also got a significant performance boost. Previously, the symbols sidebar would search for symbols in the whole directory tree even if you were only looking at a subdirectory or file, which would cause it to time out and show an error message on big monorepos. Now prefix queries like `^src/app` are better optimized and subdirectory searches are much faster and less likely to time out.
+
+## Share Notebooks with your organization
+
+Notebooks are now shareable within organizations. Previously, you could only share Notebooks with your entire instance (or share them publicly, in the case of Sourcegraph Cloud). This update allows you to create Notebooks with specific relevance to your organization, such as documentation of preferred engineering patterns or anti-patterns. Organizations' shared notebooks will now show up in their own tabs on the [Notebooks home page](https://sourcegraph.com/notebooks).
+
+<img class="blog-image" title="Notebooks sharing" alt="Image of Notebooks sharing modal." src="https://storage.googleapis.com/sourcegraph-assets/docs/images/notebooks/notebook_sharing.png">
+
+## Import and export Notebooks as markdown
```

Comment:
@iskyOS - This is also in the changelog portion. I can't remember if we include things in both the changelog section and the highlights section.

Reply:
@abschu We have once or twice before that I can find, but I don't think we've standardized on one method of doing it. Since we're prioritizing having a short changelog section with only the top items, I think it makes sense to not have redundancy